website/docs/en/blog/announcing-0.6.mdx

```diff
@@ -0,0 +1,198 @@
+
+The `module.generator` options for `css/auto` and `css/module` include `exportsOnly`, `exportsConvention`, and `localIdentName`; while the generator options for `css` include `exportsOnly` and `exportsConvention`. `exportsOnly` will replace `builtins.css.modules.exportsOnly`, `exportsConvention` will replace `builtins.css.modules.localsConvention`, and `localIdentName` will replace `builtins.css.modules.localIdentName`.
+
+In addition, there are some changes to the default values:
+
+1. The value of `exportsConvention` has changed from `'asIs'`, `'camelCaseOnly'`, etc., to `'as-is'`, `'camel-case-only'`, etc., to maintain consistency with webpack experiments css.
+2. With `namedExports: false`, it is now possible to use default exports, named exports, and namespace exports at the same time; previously, only the default export was supported:
+
+   ```js
+   // Before v0.6.0, only default export was supported
+   import classes from './index.module.css';
+
+   // Now, in addition to default export, it also supports:
+   // Namespace exports
+   import * as classes from './index.module.css';
+   // Named exports
+   import { class1, class2 } from './index.module.css';
+   // Default and named exports used together
+   import classes, { class1, class2 } from './index.module.css';
+   ```
+
+3. The default value of `namedExports` changed from `false` to `true`, meaning you'll have to use a namespace import (like `import * as classes from './style.css'`) or named import (like `import { class1 } from './style.css'`) by default, which will improve future compatibility with [native CSS modules](https://web.dev/articles/css-module-scripts). And this does not mean you have to migrate all imports at once; you can disable this behavior by setting `namedExports: false`, and since now `namedExports: false` also supports named export and namespace export, you can migrate these imports progressively.
+4. The default value of `localIdentName` has changed from `'[path][name][ext]__[local]'` in development mode and `'[hash]'` in production mode to `'[uniqueName]-[id]-[local]'` in both development and production modes, which will slightly improve the gzip compression size of the CSS output.
+5. The default value of `exportsOnly` in `target: 'node'` has changed from `false` to `true`.
+6. The default rule type for CSS has changed from `css` to `css/auto`. `css/auto` will automatically process CSS files with `.module.` or `.modules.` as infixes as [CSS Modules](https://github.com/css-modules/css-modules), consistent with [`css-loader`'s `modules.auto: true`](https://github.com/webpack-contrib/css-loader?tab=readme-ov-file#auto), which will [simplify the writing rules for using less or sass with CSS Modules](https://github.com/webpack/webpack/issues/16572).
+
+### Upgrade swc to 0.90.x
+
+Upgraded the Rust crate `swc_core` to `0.90.x`. This will affect users of the swc wasm plugin.
```

Comment:
need add a link to https://swc.rs/docs/plugin/selecting-swc-core add make a pr to swc website